src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Do we need this file here?

Reply:
As far as I understand this file is supported for importing CSS Modules. 
"This relates to import of files with .module.css,.module.scss, and .module.sass extensions."